edx_name_affirmation/api.py

```diff
@@ -75,23 +75,47 @@ def create_verified_name(
     log.info(log_msg)
 
 
-def get_verified_name(user, is_verified=False):
+def get_verified_name(
+    user, is_verified=False, verification_attempt_id=None, proctored_exam_attempt_id=None,
+):
     """
     Get the most recent VerifiedName for a given user.
 
     Arguments:
         * `user` (User object)
         * `is_verified` (bool): Optional, set to True to ignore entries that are not
           verified.
+        * `verification_attempt_id` (int): Optional
+        * `proctored_exam_attempt_id` (int): Optional
+
+    Note: `verification_attempt_id` and `proctored_exam_attempt_id` cannot be combined
```

Comment:
what if attempt_id were a small struct of (int, type)

Reply:
do you ever want to say is_verified = true AND ALSO attempt_id = something? I'd better read the ticket.

Reply:
reading the ticket it feels like there is the get something verified (don't care what)
get something pending (do we care how it's pending? that's unclear)
make a new pending item for this particular exam if needed

Reply:
@ashultz0 The main use case to get by an attempt ID would be to update the is_verified status: IE, we get a callback from the proctoring provider approving the exam attempt, in which case we'd flip the is_verified status to True.

You could technically combine the two parameters, but I can't think of a situation where it would be useful.

(Now that you bring this up, though, I'm realizing this might be outside the scope of the ticket - and Alie brings up a good point below)